### ISSUE TITLE
Connect data table headers on the Database page to locale documents for translation support

### ISSUE THREAD

**Issue by GTBarrows** [codeforboston/boston-liquor-license-tracker#496]
## Context
With the change to the Available License table and the data shown in the table. The master doc and this locale file have drifted out of alignment with the copy currently used on the production site. Currently the headers on both the "Available License" and "Recent Applications" tables are current hardcoded and aren't targeted when the website switches languages

## Task
Update the `license-availability-table.tsx` and the `recent-application-table.tsx` files and connect them with the appropriate locale documentation so that the translations can be automatic

[Master Translation Document](https://docs.google.com/document/d/1Q7XeRzKGyoxlMTgHIlnJMYFBhyfd1AZcpQR9HHeSZ-s/edit?usp=sharing)

## Note
The current translations for the "Available Licenses" table are stale and will need updating along with their locale documents. These should be hit in the same pass that covers the resources page
